src/main/java/redis/clients/jedis/params/Params.java

```diff
@@ -67,4 +67,21 @@ protected void addParam(String name) {
     params.put(name, null);
   }
 
+  @Override public String toString() {
+    ArrayList<String> paramsAsString = new ArrayList<>();
+    if (params != null) {
+      for (Entry<String, Object> param : params.entrySet()) {
+        paramsAsString.add(param.getKey());
+        Object value = param.getValue();
+        if (value != null) {
+          if (value instanceof byte[]) {
+            paramsAsString.add( SafeEncoder.encode((byte[]) value));
+          } else {
+            paramsAsString.add(value.toString());
```

Comment:
I think `paramsAsString.add(value);` is enough. Later `paramsAsString.toString();` will handle `value.toString()` anyway.